src/accessibilityStatement/AccessibilityStatement.module.css

```diff
@@ -0,0 +1,53 @@
+.pageWrapper {
+    min-height: 100vh;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+}
+
+.container {
+    margin: 20px 10px 0;
```

Comment:
I am thinking if we should not start using spacing variables from HDS now that they are relatively stable?

Reply:
Sure, I'll change these to use hds spacing.